### PR TITLE
Fedora 32

### LINKED ISSUE #449: New image for fedora 31
New image for fedora 31

### PR DESCRIPTION
**Pull request by RDIL**
Fixes #449

### CODE REVIEW

File: release/stable/fedora31/docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Docker image file that describes Fedora 31 image with PowerShell installed from Microsoft YUM Repo
+ARG fromTag=31
+ARG imageRepo=fedora
+
+FROM ${imageRepo}:${fromTag} AS installer-env
```
Comment (TravisEz13) [inline release/stable/fedora31/docker/Dockerfile:5]:
```suggestion
FROM ${imageRepo}:31 AS installer-env
```

File: release/stable/fedora31/getLatestTag.ps1 [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# return objects representing the tags we need to base the fedora image on
+
+# The versions of fedora we care about
+$shortTags = @('31')
+
+$parent = Join-Path -Path $PSScriptRoot -ChildPath '..'
+$repoRoot = Join-Path -path (Join-Path -Path $parent -ChildPath '..') -ChildPath '..'
```
Comment (TravisEz13) [inline release/stable/fedora31/getLatestTag.ps1:10]: delete this file

File: release/stable/fedora31/meta.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+    "IsLinux": true,
+    "UseLinuxVersion": false,
+    "PackageFormat": "powershell-${PS_VERSION}-1.rhel.7.x86_64.rpm",
+    "osVersion": "Fedora",
+    "SkipGssNtlmSspTests": false,
+    "tagTemplates": [
+        "#psversion#-fedora31-#tag#",
+        "fedora31-#shorttag#"
```
Comment (TravisEz13) [inline release/stable/fedora31/meta.json:9]:
```suggestion
        "#psversion#-fedora-31-#fulltag#",
        "#psversion#-fedora-#shorttag#",
        "fedora-31-#shorttag#"
```

**Review by TravisEz13 — CHANGES_REQUESTED** [review]

**Comment by anamnavi** [comment]
Given that you have removed the getLatestTag.ps1 script and we have removed fromTag from other Linux images (alpine, amazonlinux, kalilinux from this [issue](https://github.com/PowerShell/PowerShell-Docker/pull/503)) I think we could remove the fromTag from this Fedora image. (Unless there is a reason for keeping it that I'm not aware of). However, I see we have other Fedora31, Fedora30) so I could change them and Fedora32 to not use fromTag in a different PR

**Comment by RDIL** [comment]
I'll take care of it for fedora @anamnavi

File: release/stable/fedora32/meta.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
+{
+    "IsLinux": true,
+    "UseLinuxVersion": false,
+    "PackageFormat": "powershell-${PS_VERSION}-1.rhel.7.x86_64.rpm",
+    "osVersion": "Fedora 32",
+    "SkipGssNtlmSspTests": false,
+    "tagTemplates": [
+        "#psversion#-fedora-32-#fulltag#",
+        "#psversion#-fedora-32-#shorttag#",
```
Comment (TravisEz13) [inline release/stable/fedora32/meta.json:9]:
```suggestion
        "#psversion#-fedora-32",
```

File: release/stable/fedora32/docker/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,62 @@
+# Docker image file that describes Fedora 32 image with PowerShell installed from Microsoft YUM Repo
+ARG fromTag=32
+ARG imageRepo=fedora
+
+FROM ${imageRepo}:${fromTag} AS installer-env
```
Comment (TravisEz13) [inline release/stable/fedora32/docker/Dockerfile:5]:
```suggestion
FROM ${imageRepo}:32 AS installer-env
```

File: release/stable/fedora32/docker/Dockerfile [hunk 6]
```diff
@@ -0,0 +1,62 @@
+# Docker image file that describes Fedora 32 image with PowerShell installed from Microsoft YUM Repo
+ARG fromTag=32
```
Comment (TravisEz13) [inline release/stable/fedora32/docker/Dockerfile:2]:
```suggestion
```

File: release/stable/fedora32/docker/Dockerfile [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+ARG imageRepo=fedora
+
+FROM ${imageRepo}:${fromTag} AS installer-env
+
+ARG PS_VERSION=7.0.0
+ARG PACKAGE_VERSION=7.0.0
+ARG PS_PACKAGE=powershell-${PACKAGE_VERSION}-1.rhel.7.x86_64.rpm
+ARG PS_PACKAGE_URL=https://github.com/PowerShell/PowerShell/releases/download/v${PS_VERSION}/${PS_PACKAGE}
+
+ARG fromTag=32
```
Comment (TravisEz13) [inline release/stable/fedora32/docker/Dockerfile:12]:
```suggestion
```

File: release/stable/fedora32/docker/Dockerfile [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+
+# Define ENVs for Localization/Globalization
+ENV DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=false \
+    LC_ALL=en_US.UTF-8 \
+    LANG=en_US.UTF-8 \
+    # set a fixed location for the Module analysis cache
+    PSModuleAnalysisCachePath=/var/cache/microsoft/powershell/PSModuleAnalysisCache/ModuleAnalysisCache \
+    POWERSHELL_DISTRIBUTION_CHANNEL=PSDocker-Fedora-${fromTag}
```
Comment (TravisEz13) [inline release/stable/fedora32/docker/Dockerfile:20]:
```suggestion
    POWERSHELL_DISTRIBUTION_CHANNEL=PSDocker-Fedora-32
```

**Review by TravisEz13 — CHANGES_REQUESTED** [review]
see comments

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**